templates/annotate.sh

```diff
@@ -124,6 +127,72 @@ vep () {
   !{singularity_vep} vep "${args[@]}"
 }
 
+capice () {
+  local vcfPath="!{vcfPath}"
+  local vep_args=()
+  vep_args+=("--input_file" "${vcfPath}")
+  vep_args+=("--format" "vcf")
+  vep_args+=("--output_file" "${vcfPath%%.*}_prepared.vcf.gz")
+  vep_args+=("--vcf")
+  vep_args+=("--compress_output" "gzip")
+  vep_args+=("--regulatory")
+  vep_args+=("--sift" "s")
+  vep_args+=("--polyphen" "s")
+  vep_args+=("--domains")
```

Comment:
Domains field is always empty for --refseq and should be removed (also from vep function). This requires a CAPICE change.